src/main/java/com/google/cloud/spanner/pgadapter/ConnectionHandler.java

```diff
@@ -169,12 +169,23 @@ public void run() {
   }
 
   /** Called when a Terminate message is received. This closes this {@link ConnectionHandler}. */
-  public void handleTerminate() throws Exception {
+  public void handleTerminate() {
```

Comment:
Was this "throws Exception" part unnecessary or did it become obsolete due to the new changes?